yolo:
  ros__parameters:
    model_name: yolov5
    engine_path: /home/stardust/my_work/tensorrt-ros2/src/stardust_tensorrt/model/0818-sim.onnx
    conf_thres: 0.25
    nms_thres:  0.65
    class_num:  8
    class_txt: trash.txt
    width: 640
    height: 640
    rgb_topic: /color/color_raw
    depth_topic: /depth/depth_raw
    output_topic: trash_detection/detection